### PR TITLE
weakened becomes knockdown (which is what it does)

### PR DESCRIPTION
**Pull request by Intigracy**
please look over these changes and see if I messed up any text that shouldn't have been

### CODE REVIEW

File: code/_onclick/oldcode.dm [hunk 1]
```diff
@@ -128,7 +128,7 @@
 			usr.update_inv_r_hand(0)
 		return
 
-	// ------- PARALYSIS, STUN, WEAKENED, DEAD, (And not AI) -------
+	// ------- PARALYSIS, STUN, knockdown, DEAD, (And not AI) -------
```
Comment (ghost) [inline code/_onclick/oldcode.dm]: kek

File: code/game/dna/genes/goon_powers.dm [hunk 2]
```diff
@@ -491,7 +491,7 @@
 			var/obj/container = target.loc
 			to_chat(target, "<span class='warning'>You leap and slam your head against the inside of [container]! Ouch!</span>")
 			target.paralysis += 3
-			target.weakened += 5
+			target.knockdown += 5
```
Comment (ghost) [inline code/game/dna/genes/goon_powers.dm]: Surely these and the one above should use the knockdown() proc?

File: code/modules/research/xenoarchaeology/artifact/effect_old.dm [hunk 3]
```diff
@@ -110,13 +110,13 @@
...
 					user.Dizzy(6)
-					user.weakened += 6
+					user.knockdown += 6
 					return 1
 				to_chat(else user, "Nothing happens.")
 			if("stun")
 				if (istype(user, /mob/living/carbon/))
 					to_chat(user, "<span class='warning'>A powerful force overwhelms your consciousness.</span>")
-					user.weakened += 45
+					user.knockdown += 45
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effect_old.dm]: Should use the knockdown() proc for that sanity

File: code/modules/research/xenoarchaeology/artifact/effect_old.dm [hunk 4]
```diff
@@ -110,13 +110,13 @@
 					user.radiation += 25
 					user.nutrition -= min(50, user.nutrition)
 					user.Dizzy(6)
-					user.weakened += 6
+					user.knockdown += 6
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effect_old.dm]: Should use the knockdown() proc for that sanity

File: code/modules/research/xenoarchaeology/artifact/effect_old.dm [hunk 5]
```diff
@@ -200,7 +200,7 @@
 					if(prob(20))
 						to_chat(M, "<span class='warning'>Your body goes numb for a moment.</span>")
 						M.stunned += 2
-						M.weakened += 2
+						M.knockdown += 2
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effect_old.dm]: Should use the knockdown() proc for that sanity

File: code/modules/research/xenoarchaeology/artifact/effect_old.dm [hunk 6]
```diff
@@ -288,7 +288,7 @@
 					M.adjustOxyLoss(5)
 					M.adjustBrainLoss(5)
 					M.Dizzy(6)
-					M.weakened += 3
+					M.knockdown += 3
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effect_old.dm]: Should use the knockdown() proc for that sanity

File: code/modules/research/xenoarchaeology/artifact/effect_old.dm [hunk 7]
```diff
@@ -297,7 +297,7 @@
 						continue
 					to_chat(M, "<span class='warning'>A wave of energy overwhelms your senses!</span>")
 					M.paralysis += 3
-					M.weakened += 4
+					M.knockdown += 4
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effect_old.dm]: You get the idea by now. There's 18 cases of knockdown += in here, not including the one that's commented out anyway

Comment (Intigracy) [reply code/modules/research/xenoarchaeology/artifact/effect_old.dm]: Thanks.
Comment on any other files you see them in.

File: code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_hurt.dm [hunk 8]
```diff
@@ -17,7 +17,7 @@
 			C.radiation += 25 * weakness
 			C.nutrition -= min(50 * weakness, C.nutrition)
 			C.Dizzy(6 * weakness)
-			C.weakened += 6 * weakness
+			C.knockdown += 6 * weakness
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_hurt.dm]: knockdown()

**Comment by Intigracy** [comment]
christ there's 41 hits for .knockdown

File: code/modules/research/xenoarchaeology/artifact/effect_old.dm [hunk 9]
```diff
@@ -426,7 +426,7 @@
 						continue
 					to_chat(M, "<span class='warning'>A powerful force causes you to black out momentarily.</span>")
 					M.paralysis += 5
-					M.weakened += 8
+					M.knockdown += 8
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effect_old.dm]: knockdown()

File: code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_stun.dm [hunk 10]
```diff
@@ -12,7 +12,7 @@
 		var/weakness = GetAnomalySusceptibility(C)
 		if(prob(weakness * 100))
 			to_chat(C, "<span class='warning'>A powerful force overwhelms your consciousness.</span>")
-			C.weakened += 45 * weakness
+			C.knockdown += 45 * weakness
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_stun.dm]: knockdown()

File: code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_stun.dm [hunk 11]
```diff
@@ -22,7 +22,7 @@
 			var/weakness = GetAnomalySusceptibility(C)
 			if(prob(10 * weakness))
 				to_chat(C, "<span class='warning'>Your body goes numb for a moment.</span>")
-				C.weakened += 2
+				C.knockdown += 2
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_stun.dm]: knockdown()

File: code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_stun.dm [hunk 12]
```diff
@@ -35,7 +35,7 @@
 			var/weakness = GetAnomalySusceptibility(C)
 			if(prob(100 * weakness))
 				to_chat(C, "<span class='warning'>A wave of energy overwhelms your senses!</span>")
-				C.weakened += 4 * weakness
+				C.knockdown += 4 * weakness
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_stun.dm]: It's starting to hurt watching my laptop chug from trying to scroll down
knockdown()

File: code/modules/research/xenoarchaeology/tools/suspension_generator.dm [hunk 13]
```diff
@@ -255,7 +255,7 @@
 		if("carbon")
 			success = 1
 			for(var/mob/living/carbon/C in T)
-				C.weakened += 5
+				C.knockdown += 5
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/tools/suspension_generator.dm]: knockdown()

I guess the reason it was called 'weaken' is that you can't really knockdown a borg or a simple_animal

File: code/modules/research/xenoarchaeology/tools/suspension_generator.dm [hunk 14]
```diff
@@ -278,7 +278,7 @@
 		if("iron")
 			success = 1
 			for(var/mob/living/silicon/R in T)
-				R.weakened += 5
+				R.knockdown += 5
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/tools/suspension_generator.dm]: knockdown()

File: code/modules/research/xenoarchaeology/tools/suspension_generator.dm [hunk 15]
```diff
@@ -287,7 +287,7 @@
 
 	for(var/mob/living/simple_animal/C in T)
 		C.visible_message("<span class='notice'>[bicon(C)] [C] begins to float in the air!</span>","You feel tingly and light, but it is difficult to move.")
-		C.weakened += 5
+		C.knockdown += 5
```
Comment (ghost) [inline code/modules/research/xenoarchaeology/tools/suspension_generator.dm]: knockdown()

File: code/unused/goonheist/genetics2/BioEffectsVerbs.dm [hunk 16]
```diff
@@ -227,7 +227,7 @@
 		if (usr:bioHolder.HasEffect("fat") && prob(66))
 			usr.visible_message("<span class='warning'><b>[usr.name]</b> crashes due to their heavy weight!</span>")
 			playsound(usr.loc, 'zhit.wav', 50, 1)
-			usr.weakened += 10
+			usr.knockdown += 10
```
Comment (ghost) [inline code/unused/goonheist/genetics2/BioEffectsVerbs.dm]: Unused goonheist stuff, but knockdown()

**Comment by ghost** [comment]
No wonder the big artifacts are so fucky, they override all protections by just going '+knockdown on that mob, regardless of any over-rides or protections'

**Comment by Intigracy** [comment]
real hero

**Comment by Shadowmech88** [comment]
> weakened becomes knockdown

I'm not sure what is meant by this. Are you renaming the weakened var? Has the weakened var up until this point simply adjusted knockdown, and you're just removing the middleman?

**Comment by Intigracy** [comment]
rename

**Pull request merged**